code/game/machinery/cryo.dm

```diff
@@ -389,6 +438,11 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
 	updateUsrDialog()
 	return
 
+/obj/machinery/atmospherics/unary/cryo_cell/proc/Emag(mob/user as mob)
```

Comment:
Pretty certain there is, already, a generic proc for emagging : 
`emag_act(var/mob/user, var/obj/item/card/emag/E)`